client/beadledom-client-example/example-api/src/main/java/com/cerner/beadledom/client/example/model/JsonOneOffsetPaginatedListDto.java

```diff
@@ -1,5 +1,6 @@
 package com.cerner.beadledom.client.example.model;
```

Comment:
Add your name to CONTRIBUTORS.md. Thanks for the contribution!

Reply:
Added to the CONTRIBUTORS.md <img src="https://emoji.slack-edge.com/T039B0HP2/whoo/1953c486486ad14a.gif" alt="whoo" title="whoo" width="25"> and added to the CHANGELOG.md
https://github.com/cerner/beadledom/pull/116/commits/abba5dbff4b79663669c9879b5fdfa19d8c88b82#diff-4ac32a78649ca5bdd8e0ba38b7006a1e